health/REFERENCE.md

```diff
@@ -129,6 +132,32 @@ You're interested in what comes after the comma: `disk.io`. That's the name of t
 
 If you create a template using the `disk.io` context, it will apply an alarm to every disk available on your system.
 
+#### Alarm line `class`
+
+Specify the classification of the alarm or template. 
+
+Class can be used to indicate the broader area of the system that the alarm applies to. For example, under the general `Database` class, you can group together alarms that operate on various database systems, like `MySQL`, `CockroachDB`, `CouchDB` etc.
+
+```yaml
+class: Database
+```
+
+#### Alarm line `component`
+
+Component can be used to narrow down what the previous `class` value specifies for each alarm or template. Continuing from the previous example, `component` might include `MySQL`, `CockroachDB`, `MongoDB`, all under the same `Database` classification.
+
```

Comment:
Can `component` and `class` to have the exact same values? If it is not possible, we need to update the PR #10961.

Reply:
I don't recall that being a requirement... or even discussed at all for that matter. I think a requirement like that would have been explicitly stated.

Reply:
> Can component and class to have the exact same values?

Yes, "Unknown" 😄